src/Pages/Authentication/SignUp/SignUp.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import { Link } from "react-router-dom";
+import '../Css/Authentication.css';import Input from '../../../Shared/Input/Input';
+import blitheCollage from '../../../Assets/Images/Blithe-collage.jpg';
+import blitheLogo from '../../../Assets/Images/Blithe-logo.png';
+import InputField from '../../../Shared/Input/InputField';
+
+const SignUp = () =>{
+
+    const [showPassword , setShowPassword ] = useState(false);
+    const [showConfirmPassword , setShowConfirmPassword ] = useState(false);
+    const [sigUpData, setSignUpData] = useState(
+        {   
+            email:'',
+            firstName:'',
+            lastName:'',
+            password:'',
+            confirmPassword:'',
+            err:''  
+        })
+    //here err is for the validation purpose, that is yet to be implemented
+
+    //Handle show/hide password
+    const handlePassword = () =>{
+        setShowPassword(!showPassword)
+    }
+
```

Comment:
You can use setShowPassword(showPassword => !showPassword)

Reply:
ok!